std/traits.d

```diff
@@ -1480,6 +1484,186 @@ private FunctionAttribute extractAttribFlags(Attribs...)()
     return res;
 }
 
+private enum bool isSomeStringInfer(alias T) = isSomeString!(typeof(T));
+
+/**
+Checks whether a function has the given attributes attached.
+
+Params:
+    func = function to check
+    attributes = variadic number of function attributes as strings
+
+Returns:
+    `true`, if the function has the list of attributes attached and `false` otherwise.
+
+See_Also:
+    $(LREF functionAttributes)
+*/
+/// ditto
```

Comment:
oh that was just a relict from trying to have different overloads. Thanks for spotting that :)